kube/resources/worker.yaml

```
@@ -94,6 +94,11 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.namespace
+            - name: JOB_KUBE_SERVICE_ACCOUNT
+              valueFrom:
+                configMapKeyRef:
+                  name: airbyte-env
+                  key: LJOB_KUBE_SERVICE_ACCOUNT
```

The L is probably also a typo
```suggestion
                  key: JOB_KUBE_SERVICE_ACCOUNT
```

---

This file is no longer included in the new PR https://github.com/airbytehq/airbyte-platform/pull/177/